zipline/gens/brokers/rh/robinhood.py

```diff
@@ -0,0 +1,1147 @@
+"""Robinhood.py: a collection of utilities for working with Robinhood's Private API """
```

Comment:
Okay. I just realized that this code has been copy pasted from https://github.com/Jamonek/Robinhood/blob/master/Robinhood/Robinhood.py.

Please don't copy it here, but rather use it as external library.
If you have modifications in it we can create our own fork, but we should not compound libraries into zipline-live.